samples/bluetooth/hci_rpmsg/prj.conf

```diff
@@ -16,3 +16,4 @@ CONFIG_BT_HCI_RAW_RESERVE=1
 CONFIG_ASSERT=y
 CONFIG_DEBUG_INFO=y
 CONFIG_EXCEPTION_STACK_TRACE=y
+CONFIG_BT_CTLR_TX_PWR_DYNAMIC_CONTROL=y
```

Comment:
why enable for all applications that would use hci_rpmsg? instead provide a separate conf file to build hci_rpmsg for hci_pwr_ctrl sample.